### PR TITLE
Problem: Missing SendEip155Transaction on Metaverse demo (fix cronoslabs/play-unreal-plugin#296)

### PR DESCRIPTION
**Pull request by damoncro**

### CODE REVIEW

File: Metaverse/Source/Metaverse/Public/NPC/SendEthTransaction.h [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// Fill out your copyright notice in the Description page of Project Settings.
```
Comment (tomtau) [inline Metaverse/Source/Metaverse/Public/NPC/SendEthTransaction.h:1]: Should this be updated / removed?

**Review by tomtau — APPROVED** [review]
Seems ok, just some files have the default copyright message comment

File: Metaverse/Source/Metaverse/Public/NPC/SendEthTransaction.h (hunk 1 above)
Comment (damoncro) [reply Metaverse/Source/Metaverse/Public/NPC/SendEthTransaction.h:1]: oh right

**Comment by damoncro** [comment]
The CI error is because https://github.com/cronos-labs/play-unreal-plugin/pull/295 is not yet merged.

**Review by leejw51crypto — APPROVED** [review]
lgtm

**Comment by damoncro** [comment]
Please help merge https://github.com/cronos-labs/play-unreal-plugin/pull/295 if no issues, so that this PR can continue. 
@leejw51crypto

**Review by rarcifa — APPROVED** [review]

**Pull request merged**